src/main/scala/com/tresata/spark/scalding/CascadingRDD.scala

```diff
@@ -20,7 +20,7 @@ import org.apache.spark.rdd.RDD
 import org.apache.spark.util.TaskCompletionListener
 
 class RichSource(val source: Source) extends AnyVal {
-  def spark(implicit sc: SparkContext): FieldsRDD = CascadingRDD(sc, source)
+  def spark(implicit sc: SparkContext, conf: Configuration): FieldsRDD = CascadingRDD(sc, source, conf)
```

Comment:
if we do:
```
def spark(implicit sc: SparkContext, conf: Configuration = new Configuration): FieldsRDD = ...
```
then it will also work outside of SparkJob